packages/wallets/src/components/WalletNoWalletFoundState/WalletNoWalletFoundState.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import NoWalletIcon from '../../public/images/no-wallet.svg';
+import { WalletButton, WalletText } from '../Base';
+import './WalletNoWalletFoundState.scss';
+
+const WalletNoWalletFoundState = () => {
+    const history = useHistory();
+    return (
+        <div className='wallets-no-wallet-found-state'>
+            <NoWalletIcon height={300} width={300} />
+            <div className='wallets-no-wallet-found-state-content'>
+                <h3 className='wallets-no-wallet-found-state-content__title'>You have no wallet account 🐣</h3>
+                <p className='wallets-no-wallet-found-state-content__description'>
+                    Disable the <span>next_wallet</span> feature flag to see Trader&apos;s Hub{' '}
```

Comment:
feels like we shouldn't leave messages for developers within our codebase, to avoid displaying it to customers accidentally

Reply:
@wojciech-deriv This is only for internal use. We only show it if next_wallet feature flag is enable, which only being used by us developers and testers. Later on, we can show 404 page based on backend flag, if user havent migrated to wallet.